### PR TITLE
Hide Article

### PR DESCRIPTION
**Pull request by khaledkz**
Add ability to hide article, change, update the form to take the value for the selected category, update api single category and findarticle to hide article

### CODE REVIEW

File: server/views/index.hbs [hunk 1]
```diff
@@ -8,11 +8,9 @@
     </tr>
   </thead>
   <tbody>
-    {{#each articles}}
+    {{#each articles}}{{#if this.visible}}
```
Comment (kabaros) [inline server/views/index.hbs]: @kharazzoro .. `views/index.hbs` and `admin-list-articles` both have a list of articles and the code is duplicated. Could you DRY it -maybe move the list code to a partial and use it in both views

File: server/routes/api.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ router.get('/articles', function (req, res, next) {
             res.json(articles)
         }
     };
-    articleClient.findArticles({}, callBack)
+    articleClient.findArticles({ 'visible': true }, callBack)
```
Comment (kabaros) [inline server/routes/api.js]: no need for `'` around `visible` since it's one word (and it doesn't have dashes or spaces)

**Review by kabaros — CHANGES_REQUESTED** [review]
very good PR - just some small changes.

File: server/views/index.hbs (hunk 1 above)
Comment (khaledkz) [reply server/views/index.hbs]: Do I need to use partials, and does it work int hbs same in handalebars

**Comment by kabaros** [comment]
It should .. give it a try. If it doesn't, then you can use the same view
and have a different layout defined when it's home page.

sent from my phone

**Review by kabaros — APPROVED** [review]

**Pull request merged**